types/variables/variables.go

```diff
@@ -25,8 +25,8 @@ import (
 // This file repeats the same content many times in order to make access
 // efficient for seclang and transactions
 
-// VariablesCount contains the number of existing variables
-const VariablesCount = 87
+// Count contains the number of existing variables
+const Count = 87
```

Comment:
The reason I used VariablesCount instead of Count is because Count can be easily mistaken as a valid seclang variable, what do you think?

Reply:
I think the ```Count``` is under the variables package, it would be cumbersome to use ```VariablesCount```.

Reply:
I agree with you, what about moving VariablesCount to the types package? It still bugs me under variables

Reply:
It's ok with moving the ```VariablesCount``` to the types package so that it will not appear wordy.

Reply:
I discard the change here, I'll revert.